src/test/resources/templates/yetAnotherTestFlow/subflowAddItemPage2.html

```diff
@@ -3,82 +3,82 @@
 <head th:replace="~{fragments/head :: head(title='Subflow Page2')}"></head>
 <body>
 <div class="page-wrapper">
-  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
-  <section class="slab">
-    <div class="grid">
-      <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block
-            th:replace="~{fragments/cardHeader :: cardHeader(header='Subflow Page', subtext='This is a page within a subflow')}"/>
-        <!-- Test that the currentSubflowItem is passed in by the controller: -->
-        <div th:text="${currentSubflowItem.firstNameSubflowPage2}"></div>
-        <div>
-          <th:block
-              th:replace="~{fragments/inputs/text ::
+    <div th:replace="~{fragments/toolbar :: toolbar}"></div>
```

Comment:
There are no actual changes in this file, other then format ones. I can revers this, but since we are moving to these new spacings anyways, I'll just leave this.